index.js

```diff
@@ -17,6 +17,16 @@ module.exports = async (entry, { minify = true } = {}) => {
       filename: "out.js",
       libraryTarget: "commonjs2"
     },
+    externals: (context, request, callback) => {
+      resolve(request, { basedir: context, preserveSymlinks: true }, (err) => {
```

Comment:
Another way to do this might be to put this into a `resolveId` plugin hook (a local inline plugin). `null` returns from resolveId are treated as externals. In the plugin hook, you'd then have access to the global resolver through `this.resolveId` so that it exactly matches the internal resolve implementation (no need to worry about divergence between `require('resolve')` and rollup resolve.

Reply:
This is using webpack though 🕵️

Reply:
Ah sorry, missed that!

Reply:
I assume something similar can be done using webpack, but this was the least time-consuming way to make it work 👍